bsm_helper/src/bsm_helper.cpp

```diff
@@ -0,0 +1,40 @@
+#include "bsm_helper/bsm_helper.h"
+
+namespace BSMHelper{
+
+
+    std::string BSMHelper::bsmIDtoString(std::vector<uint8_t, std::allocator<uint8_t>> id)
```

Comment:
Please add a size check to this method which verifies the input array has a length that fits within the requirements of a BSM. Throw an exception in this case.